Portal/View/UIKit/Renderers/Utils/MessageHandler.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  MessageHandler.swift
+//  Portal
+//
+//  Created by Argentino Ducret on 8/23/17.
+//  Copyright © 2017 Guido Marucci Blas. All rights reserved.
+//
+
+protocol MessageHandler where Self : UIControl {
```

Comment:
I like what you did with this protocol, my suggestion would be to drop the `dispatcherKey` in `getDispatcher` and also avoid the `mailboxKey` in `getMailbox`